src/main.rs

```diff
@@ -269,16 +270,21 @@ impl EventHandler for Handler {
 async fn main() {
     tracing_subscriber::fmt::init();
 
-    // Configure the client with your Discord bot token in the environment.
-    let token = env::var("DISCORD_TOKEN").expect("Expected a token in the environment");
-
     let framework = StandardFramework::new();
-    let username =
-        env::var("SPOTIFY_USERNAME").expect("Expected a Spotify username in the environment");
-    let password =
-        env::var("SPOTIFY_PASSWORD").expect("Expected a Spotify password in the environment");
-    let user_id =
-        env::var("DISCORD_USER_ID").expect("Expected a Discord user ID in the environment");
+
+    let config = match Config::new() {
+        Ok(config) => config,
+        Err(error) => {
+            println!("Couldn't read config");
+            if let MissingField(f) = error.kind {
+                println!("Missing field: 'AOEDE_{}'", f.to_uppercase());
```

Comment:
I think the `AOEDE_` should be removed in the message.

Reply:
Probably should have been, thx!